.gitlab/build_and_test.sh

```diff
@@ -88,7 +88,6 @@ then
         mkdir "${buildcache}"
     fi
 
-    chmod g+rwx -R $buildcache
```

Comment:
Should we note in the dev docs that this needs to be done manually (and similarly for other directories under the shared `ci` directory)?